diff --git a/role_theme_switcher.module b/role_theme_switcher.module
index f83a3c5..b395975 100644
--- a/role_theme_switcher.module
+++ b/role_theme_switcher.module
@@ -26,7 +26,7 @@ function role_theme_switcher_custom_theme()
  */
 function role_theme_switcher_menu()
 {
-  $items['admin/user/themes'] = array(
+  $items['admin/people/permissions/themes'] = array(
     'type'            => MENU_NORMAL_ITEM,
     'title'           => 'Role theme switcher',
     'description'     => 'Settings for role theme switcher.',
